app/Controllers/feedController.php

```diff
@@ -731,8 +740,9 @@ public function reloadAction() {
 		$entryDAO2 = FreshRSS_Factory::createEntryDao();
 
 		foreach ($entries as $entry) {
-			$entry->loadCompleteContent(true);
-			$entryDAO2->updateEntry($entry->toArray());
+			if ($entry->loadCompleteContent(true)) {
```

Comment:
Minor unrelated error fix, sorry :-)

Reply:
When the CSS Path for full content retrieval was empty, the *reload* function would cause errors